nucypher/cli/painting/staking.py

```diff
@@ -84,6 +85,12 @@ def paint_stakes(emitter: StdoutEmitter,
     emitter.echo(f"Staker {staker.checksum_address} ════", bold=True, color='red' if missing else 'green')
     worker = staker.worker_address if staker.worker_address != NULL_ADDRESS else 'not bonded'
     emitter.echo(f"Worker {worker} ════", color='red' if staker.worker_address == NULL_ADDRESS else None)
+    if stakeholder and stakeholder.worker_data:
+        worker_data = stakeholder.worker_data.get(staker.checksum_address)
+        if worker_data:
+            emitter.echo(f"\t public address: {worker_data['publicaddress']}")
+            if worker_data.get('nucypher version'):
```

Comment:
should we use dictionary keys without spaces?

Reply:
Pylint is fine with it.
https://stackoverflow.com/questions/13474957/spaces-in-python-dictionary-keys
Is there some coding spec document to which we have all agreed that I'm not aware of?